packages/node/src/plugins/DynamicFilesystemChunkLoadingRuntimeModule.ts

```diff
@@ -184,9 +184,9 @@ class DynamicFilesystemChunkLoadingRuntimeModule extends RuntimeModule {
         this.runtimeRequirements.has(RuntimeGlobals.onChunksLoaded),
       ),
       '',
-      this.runtimeRequirements.has(RuntimeGlobals.ensureChunkHandlers)
-        ? generateLoadScript(runtimeTemplate)
-        : '// no remote script loader needed',
+      // this.runtimeRequirements.has(RuntimeGlobals.ensureChunkHandlers)
```

Comment:
old code that is already avaliable via loadEntryString in sdk, require.l should exist in bundler runtime or equivalent